docs/TXSPEC.md

```diff
@@ -26,9 +26,9 @@ Transactions passed to the Ledger device will be in the following format. The Le
 ```json
 {
   "account_number": "123",
-  "chain_id": "cosmoshub-4",
+  "chain_id": "panacea-3",
   "fee": {
-    "amount": [{"amount": "4000", "denom": "uatom"}, ...],
+    "amount": [{"amount": "4000", "denom": "umed"}, ...],
     "gas": "40000"
```

Comment:
It seems that our chain has set a fee of 5umed and 200000 gas.

Reply:
This document is only for an example. The setting of 5umed and 200000gas is configured in each network. So, this ledger implementation doesn't need to care about those settings.

Reply:
Specifically, the ledger app always use configurations that are specified by `--fees`, `--gas` and `--gas-price`.

Reply:
I got it. Thank you